public/404.html

```diff
@@ -31,6 +31,17 @@
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-128295598-2"></script>
```

Comment:
@mikedaul Looks like we need to use that gaID variable here too...

Reply:
crap - thanks for finding that